### DISCUSSION TITLE
New Users and assign to Teams

### DISCUSSION THREAD

**Discussion by MarkusWz** [kimai/kimai#2800] [Questions & Answers] [answered]
I detected the following:
We have two locations with several teams each.
There is one person in each location taking care about new users put to kimai, but also taking care to export data from kimai on a monthly base. (I call them time tracking responsible = TTR from now on).

When a time tracking responsible puts a new user in kimai, this user is not assigned to any team as a starting point.
The TTR has now no posibility to see the user just put in kimai as not assigned to the TTR team unless I give the permissions "view_all_data".

But giving these permission to the TTR means that in the export of all users in a certain location is not possible until I select all users manually in the export button.

Is there any possibility to give a role the permissions to see all users not using the option "view_all_data"?

**Comment by kevinpapst** [comment, marked as answer]
There is an open issue, which you can vote for: assign new users to default team.
I guess that would solve your problem, as the new user could become automatically member of a "all users" team.
This "all users" team would then have TTR as teamlead.

**Reply by MarkusWz** [reply]
Yes, this would solve the problem, as with the latest version it is also possible to include more than one team lead to a group. So all TTR would be team leads for the group "all users" and would see them then.

**Reply by kevinpapst** [reply]
Good. Still looking for a company to sponsor that feature 😁 if someone is interested, send me a email.

**Reply by MarkusWz** [reply]
Where do I find this issue ... via search I don't see it right now to vote for ...

**Reply by MarkusWz** [reply]
I guess you mean this here, right:
https://github.com/kevinpapst/kimai2/issues/1845

**Reply by kevinpapst** [reply]
I thought there is a dedicated issue, but maybe it is just in my head. Go ahead a create a new issue, thats better.

**Reply by MarkusWz** [reply]
I added this: https://github.com/kevinpapst/kimai2/issues/2802
